azure_functions_worker/loader.py

```diff
@@ -91,3 +93,32 @@ def load_function(name: str, directory: str, script_file: str,
             f'present in {rel_script_path}')
 
     return func
+
+
+@attach_message_to_exception(
+    expt_type=ImportError,
+    message=f'Troubleshooting Guide: {MODULE_NOT_FOUND_TS_URL}'
+)
+def index_function_app(directory: str) -> typing.List[Function]:
+    function_path = os.path.join(directory, SCRIPT_FILE_NAME)
```

Comment:
Don't import it directly - as we have to keep it open to a future `functions_setup.cfg` file.